src/ripple/rpc/handlers/AccountOffers.cpp

```diff
@@ -24,6 +24,19 @@
 
 namespace ripple {
 
+void appendOfferJson (std::shared_ptr<SLE const> const& offer,
```

Comment:
I could buy that the explicitness makes the code more immediately accessible. Those particular aliases, nested or otherwise, didn't seem gratuitous though, and there was a family of *::ref and *::pointer ones with uniform (if a little surprising) semantics and which were pretty easy to memorise. I imagine they came about quite organically.

In any case, if I was writing enough code that it annoyed me writing verbose type definitions out, I could just use editor snippets with some alias name as the trigger text eh?